.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+ + +sudo: false
+ +
+ + +node_js:
+ +   - '5'
+ +   - '4'
+ +
+ + cache:
+ +   directories:
+ +   - node_modules
+ +
+ + after_script:
+ +     npm run coverage
```

Comment:
is _coverage_ defined as script in package.json? if not, you either add it there or remove this _after_script_ from here